### DISCUSSION TITLE
JPEG XL And Google's War Against It

### DISCUSSION THREAD

**Discussion by giscus** [DeclanChidlow/vale.rocks#22] [Comments Archive]
# JPEG XL And Google's War Against It

My look into the very promising JPEG XL and why it hasn't seen the adoption it deserves as an innovative new file format.

https://vale.rocks/posts/jpeg-xl-and-googles-war-against-it

**Comment by Hefaistos68** [comment]
SixLabors image library is working on jpgxl support in native c#.

**Reply by DeclanChidlow** [reply]
I can find some light discussion about it being considered, but nothing saying that any work has been completed nor that it definitely will be implemented. Hopefully they do move forward with it.

**Reply by Hefaistos68** [reply]
Nope, nothing ready yet. It's quite some work to rewrite the whole jpgxl library from c++ to c#.
But it's underway.

**Reply by DeclanChidlow** [reply]
I can imagine it'd be quite the undertaking, but I'm glad to hear it's happening. Can't wait to see what comes of it!

**Comment by loeggggg** [comment]
the law of nature, so beautiful
